www/content/workstation/config_rb.md

```diff
@@ -137,51 +107,39 @@ This configuration file has the following settings:
     ```
 
 `data_bag_encrypt_version`
-
-:   The minimum required version of data bag encryption. Possible
-    values: `1` or `2`. When all of the machines in an organization are
-    running Chef Client 11.6 (or higher), it is recommended that this
-    value be set to `2`. For example:
+: The minimum required version of data bag encryption. Possible values: `1` or `2`. When all of the machines in an organization are running Chef Client 11.6 (or higher), it is recommended that this value be set to `2`. For example:
 
     ``` ruby
     data_bag_encrypt_version 2
     ```
 
 `fips`
+: Allows OpenSSL to enforce FIPS-validated security during a Chef Infra Client run. Set to `true` to enable FIPS-validated security.
+FIPS support is available in Chef Client versions 12.8 and above.
+The following operating systems are supported:
 
-:   Allows OpenSSL to enforce FIPS-validated security during a Chef
-    Infra Client run. Set to `true` to enable FIPS-validated security.
-
-    FIPS support is available in Chef Client versions 12.8 and above.
-    The following operating systems are supported:
-
-    -   Red Hat Enterprise Linux
-    -   Oracle Enterprise Linux
-    -   CentOS
-    -   Windows
+    - Red Hat Enterprise Linux
+    - Oracle Enterprise Linux
+    - CentOS
+    - Windows
 
     Support for FIPS was introduced in Chef Server version 12.13. The
     following operating systems are supported:
 
-    -   Red Hat Enterprise Linux
-    -   Oracle Enterprise Linux
-    -   CentOS
+    - Red Hat Enterprise Linux
```

Comment:
All these OS releases can go away. We're talking about the config.rb so it doesn't matter what platforms you can install Chef Infra Server on.